aptos-move/block-executor/src/executor.rs

```diff
@@ -699,6 +715,16 @@ where
                 true,
             );
 
+            let res = match res {
+                ExecutionStatus::Success(output) => ExecutionStatus::Success(output),
+                ExecutionStatus::SkipRest(output) => ExecutionStatus::SkipRest(output),
+                ExecutionStatus::Abort(err) => ExecutionStatus::Abort(Error::UserError(err)),
```

Comment:
I don't think this is correct, ExecutionStatus::Abort means nothing gets committed, the whole execution returns error

Reply:
Not sure what you mean, but we will have a way to handle remote execution error in cross shard soon. This doesn't change the existing behavior as we return `UserError` -  same as previously.

Reply:
note this is not a move abort, basically this invalidates the whole block execution (including previous ones that already goes through commit hooks),

Reply:
Hmm I see.. so how do we handle this in sharded execution? Just abort the entire sharded execution I guess? When does this happen?